verification/curator-service/ui/src/util/display-date.ts

```diff
@@ -5,7 +5,7 @@
  * locale (e.g. day-month-year vs. month-day-year), but set the date timezone
  * to UTC.
  */
-export default function displayDate(date: Date) {
+export default function displayDate(date: Date): string {
```

Comment:
I don't think we're still using this anywhere. Remove?

Reply:
true, removed.